enrolment-ui/package.json

```diff
@@ -3,44 +3,63 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "axios": "^0.18.1",
-    "country-region-data": "^1.4.2",
-    "i18next": "^10.0.7",
-    "i18next-xhr-backend": "^1.5.0",
-    "js-cookie": "^2.1.4",
-    "lodash": "^4.17.13",
-    "react": "^16.0.0",
-    "react-bootstrap": "^0.31.5",
-    "react-bootstrap-table": "^4.1.1",
-    "react-dom": "^16.0.1",
-    "react-google-login": "^3.0.1",
-    "react-i18next": "^6.1.0",
-    "react-redux": "^5.0.6",
+    "axios": "^0.21.0",
+    "country-region-data": "^1.7.0",
+    "i18next": "^19.8.4",
+    "i18next-http-backend": "^1.0.21",
+    "js-cookie": "^2.2.1",
+    "lodash": "^4.17.20",
+    "react": "^17.0.1",
+    "react-bootstrap": "^1.4.0",
+    "react-bootstrap-table": "^4.3.1",
+    "react-dom": "^17.0.1",
+    "react-google-login": "^5.1.25",
+    "react-i18next": "^11.8.1",
+    "react-redux": "^7.2.2",
     "react-router": "^4.2.0",
```

Comment:
holding back on router stuff as it wasn't vulnerable, and requires extra refactoring work.